src/main/java/seedu/address/model/house/Block.java

```diff
@@ -31,6 +31,9 @@ public Block(String block) {
      * Returns true if a given string is a valid block number.
      */
     public static boolean isValidBlock(String test) {
+        if (test.equals("N/A")) {
+            return true;
+        }
```

Comment:
Can consider just having an empty value if there's no block.
And then just show "N/A" on display if the value is empty.